src/app/components/mnoe-organizations-list/mnoe-organizations-list.coffee

```diff
@@ -1,32 +1,73 @@
 #
 # Mnoe organizations List
 #
-@App.directive('mnoeOrganizationsList', ($filter, $log, MnoeOrganizations, MnoeAdminConfig, MnoeCurrentUser) ->
+@App.directive('mnoeOrganizationsList', ($filter, $translate, MnoeOrganizations, MnoeAdminConfig, MnoeCurrentUser) ->
   restrict: 'E'
   scope: {
     list: '='
   },
   templateUrl: 'app/components/mnoe-organizations-list/mnoe-organizations-list.html',
-  link: (scope, elem, attrs) ->
-
-    # Widget state
-    scope.state = attrs.view
+  link: (scope, elem) ->
 
     # Variables initialization
     scope.organizations =
       search: ''
+      sortAttr: 'created_at'
       nbItems: 10
       page: 1
       pageChangedCb: (nbItems, page) ->
         scope.organizations.nbItems = nbItems
         scope.organizations.page = page
         offset = (page  - 1) * nbItems
-        fetchOrganizations(nbItems, offset)
+        fetchOrganizations(nbItems, offset, scope.organizations.sortAttr)
+
+    $translate(["mnoe_admin_panel.dashboard.organization.account_frozen_state",
+      "mnoe_admin_panel.dashboard.organization.widget.list.table.creation",
+      'mnoe_admin_panel.dashboard.organization.widget.list.table.name'
+      'mnoe_admin_panel.dashboard.organization.widget.list.table.revenue',
+      'mnoe_admin_panel.dashboard.organization.widget.list.table.margin'
+      'mnoe_admin_panel.dashboard.organization.widget.list.table.currency']).then((locale) ->
+        basicFields = [
```

Comment:
Is there a way to make this a bit more readable/maintanable? 
eg: better indentation, splitting into different parts?

/cc @alexnoox

Reply:
Yes, I think the readability could be improved, using comments and spaces.